openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/ListFilter.java

```diff
@@ -103,12 +105,14 @@ public String getTestSuiteFQNCondition() {
     return testSuiteName == null ? "" : getFqnPrefixCondition(null, testSuiteName, "testSuite");
   }
 
-  private String getDomainCondition() {
+  private String getDomainCondition(String tableName) {
     String domainId = getQueryParam("domainId");
     return domainId == null
         ? ""
-        : "(id in (SELECT toId FROM entity_relationship WHERE fromEntity='domain' AND fromId=:domainId AND "
-            + "relation=10))";
+        : String.format(
+            "(%s in (SELECT entity_relationship.toId FROM entity_relationship WHERE entity_relationship.fromEntity='domain' AND entity_relationship.fromId IN (%s) AND "
+                + "relation=10))",
+            nullOrEmpty(tableName) ? "id" : String.format("%s.id", tableName), domainId);
```

Comment:
@mohityadav766 should we keep the filters as parametrized filters (i.e. `:<something>`) or is it sanitized upstream?